homeassistant/components/flux_led/config_flow.py

```diff
@@ -204,12 +205,19 @@ async def async_step_discovery_confirm(
             step_id="discovery_confirm", description_placeholders=placeholders
         )
 
+    @callback
+    def _async_abort_if_host_configured(self, host: str) -> None:
+        """Abort if the host is already configured unless its an ignored entry."""
+        for entry in self._async_current_entries(include_ignore=False):
+            if entry.data.get(CONF_HOST) == host:
+                raise AbortFlow("already_configured")
```

Comment:
Can this helper work? I am using similar in `webostv`

https://github.com/home-assistant/core/blob/691cb378a07c32dd424b2d997fef463bcbb9d32b/homeassistant/config_entries.py#L2861

https://github.com/home-assistant/core/blob/260121720945fdf1fc49dd287f28ead168a25223/homeassistant/config_entries.py#L2796-L2812

Reply:
That makes more sense. No changes were actually needed. I turned out I had some cruft in core.config_entires that was the source of the issue.

I updated this PR to be a code quality improve to add a test instead.